integration_tests/dockerfiles/initia/build.sh

```diff
@@ -3,7 +3,7 @@ echo "Building initia-test docker image"
 DIR="$(dirname $0)"
 cd $DIR
 VERSION=$(cat ../../package.json | jq -r '.version')
-git clone https://github.com/initia-labs/initia -b v0.2.15
+git clone https://github.com/initia-labs/initia 
```

Comment:
I don't see any calls to git checkout in Dockerfile. Is it okay that we always build from default branch, without pinning to any concrete version?

Reply:
till the mainnet I'd prefer to have it from main branch